backend/theme/serializers.py

```diff
@@ -39,8 +39,8 @@ def serialize_theme(theme: ThemeConfig) -> dict:
         'description': theme.description,
         'headingFontUrl': theme.heading_font_url,
         'bodyFontUrl': theme.body_font_url,
-        'logoUrl': join(settings.MEDIA_URL, str(theme.logo_image.file)) if theme.logo_image else None,
-        'backgroundUrl': join(settings.MEDIA_URL, str(theme.background_image.file)) if theme.background_image else None,
+        'logoUrl': f'{settings.BASE_URL.strip("/")}/{settings.MEDIA_URL.strip("/")}/{str(theme.logo_image.file)}' if theme.logo_image else None,
+        'backgroundUrl': f'{settings.BASE_URL.strip("/")}/{settings.MEDIA_URL.strip("/")}/{str(theme.background_image.file)}' if theme.background_image else None,
```

Comment:
Can't you use the `serialize_image` method like in the other PRs of today?

https://github.com/Amsterdam-Music-Lab/MUSCLE/blob/38669ad060e69c85fcc43f1676ac8beebe5c5641/backend/image/serializers.py#L8-L13

Reply:
Eventually, that's 100% the better way, yes. For now, the frontend changes involved to adapt to the objects here are kind of daunting (especially to use the href defined in the images).